src/app/index.jsx

```diff
@@ -71,24 +52,189 @@ const App = () => {
 		}
 	}, [todos, initialTodos]);
 
-	
 	// App UI
 	return (
 		<Fragment>	
-			<Header />
 			<main className="main">
-				<TodoCreate   addTodo={addTodo} />
-				<TodoList
-					completedTodos={completedTodos}
-					totalTodos={totalTodos}
-					showTodos={showTodos}
-					checkTodo={checkTodo}
-					deleteTodo={deleteTodo}
-					deleteTodoCompleted={deleteTodoCompleted}
-					setFilterTodo={setFilterTodo}
-					todos = {todos}
-					saveTodos = {saveTodos}
-				/>
+				<section className='sectionDates'>
+					<div className="sectionDates-header">
+						<div className="header-logo"></div>
+						<h3>Nicarao Agency</h3>
+					</div>
+					<div className="section-reminders">
+						<div className="reminders-title">
+							<h4>Weekly Pinned</h4>
+							<button>View All</button>
+						</div>
+						<div className="reminders-container">
+
+
+							<div className='reminder reminder1'>
```

Comment:
Seguimos con problemas en la consistencia del codigo. arriba veo que estas usuando comillas dobles y aca sencillas. ¿Por qué?

```suggestion
							<div className="reminder reminder1">
```